src/content/docs/products/splashkit/SplashKit Tutorials/0-overview.mdx

```diff
@@ -1,9 +1,41 @@
 ---
-title: Overview
+title: Tutorials Overview
+description: The SplashKit tutorials team works on the creation of tutorials using the SplashKit library.
 sidebar:
   order: 0
 ---
 
 import { Card, LinkCard, CardGrid, Icon } from "@astrojs/starlight/components";
 
-Placeholder page.
+The SplashKit tutorials team works on the creation of tutorials using the SplashKit library. The team is responsible for creating tutorials that cover a wide range of topics, from Graphics, Interfaces, Networking and Raspberry Pis. The tutorials are designed to help users learn how to use SplashKit and to provide them with the knowledge they need to create their own projects. Each tutorial contains an explanation of the topic, the steps involved, as well as code snippets done in C#, C++, Python and Pasal.
+
+## Tutorial Topics
+
+The current list of tutorial topics includes:
```

Comment:
This list will grow as work continues, requiring this page to be updated every time a new tutorial is added (or removed). Recommend pointing to the current list of tutorials/guides (https://splashkit.io/guides/).

Reply:
Most tutorials will fit in under those categories, if they're new areas they will need to be added into the tutorial list and tutorial reviews so updating this page would only be an additional step.

Reply:
I am still of the belief that this needs to be reconsidered. If the end goal is to have tutorials that cover all topics then why have an exhaustive list now? For example, I am working on Physics and Vectors at the moment. As this doesn't exist in this list, does that mean that I can't work on it? Does the responsibility of updating this page then fall upon the contributing member? Why add multiple locations to update the list of current topics when we can use the live list (i.e. the one at splashkit.io) as a single source of truth. This is double handling work and leaving us vulnerable to having out of date content.